src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -69,7 +68,8 @@ import { ConfirmDialogType } from '@/interfaces'
 export default class AddNameTranslation extends Vue {
   // Refs
   $refs!: {
-    confirmTranslationDialog: ConfirmDialogType
+    confirmTranslationDialog: ConfirmDialogType,
+    nameTranslationForm: HTMLFormElement
```

Comment:
We already created a type for forms (FormType) that exposes `validate()`. I think that would be better than using a generic type that doesn't.

On the positive side, it looks like HTMLFormElement would be a great base type for FormType. (It current extends Vue.) Why don't you update it? It would be a nice improvement to our framework/architecture :)

Reply:
Done